ocaml-eglot.el

```diff
@@ -69,6 +69,13 @@ Otherwise, `merlin-construct' only includes constructors."
   :group 'ocaml-eglot
   :type 'string)
 
+(defcustom ocaml-eglot-open-window-strategy 'smart
+  "Defines window opening strategy."
+  :group 'ocaml-eglot
+  :type '(choice (const :tag "Open a new window only if the target differs" smart)
+                 (const :tag "Always open in a new window" new)
+                 (const :tag "Always open in the current window" current)))
+
```

Comment:
Do you expect this setting to be reused by other requests ? 
If not we should say in the name and doc that it is specific to locate requests.

There are other requests that might use a similar setting, but users might want different values for each ?

Reply:
TBH I don't know. But I guess it is nice to keep configuration behaviour _generic_ ?

Reply:
Sound good, we can always introduce specialized versions that shadow this setting if we feel like we need it.